05homework.py

```diff
@@ -0,0 +1,53 @@
+""" Homework 05 list of tuples.
+
+# Given list of tuples (people list -> name, surname, age, profession,
+#     City location)
+# 1 - Add your new record with similar random data to the beginning of
+#     the given list
+# 2 - In modified list swap elements with indexes 1 and 5  (1<->5) and
+#     print result
+# 3 - check condition that all people in modified list with records
+#     indexes 6, 10, 13 have age >=30 and print result
+
+people_records = [
+    ('John', 'Doe', 28, 'Engineer', 'New York'),
+    ('Alice', 'Smith', 35, 'Teacher', 'Los Angeles'),
+    ('Bob', 'Johnson', 45, 'Doctor', 'Chicago'),
+    ('Emily', 'Williams', 30, 'Artist', 'San Francisco'),
+    ('Michael', 'Brown', 22, 'Student', 'Seattle'),
+    ('Sophia', 'Davis', 40, 'Lawyer', 'Boston'),
+    ('David', 'Miller', 33, 'Software Developer', 'Austin'),
+    ('Olivia', 'Wilson', 27, 'Marketing Specialist', 'Denver'),
+    ('Daniel', 'Taylor', 38, 'Architect', 'Portland'),
+    ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix')]
+
+"""
+
+people_records = [
+    ('John', 'Doe', 28, 'Engineer', 'New York'),
+    ('Alice', 'Smith', 35, 'Teacher', 'Los Angeles'),
+    ('Bob', 'Johnson', 45, 'Doctor', 'Chicago'),
+    ('Emily', 'Williams', 30, 'Artist', 'San Francisco'),
+    ('Michael', 'Brown', 22, 'Student', 'Seattle'),
+    ('Sophia', 'Davis', 40, 'Lawyer', 'Boston'),
+    ('David', 'Miller', 33, 'Software Developer', 'Austin'),
+    ('Olivia', 'Wilson', 27, 'Marketing Specialist', 'Denver'),
+    ('Daniel', 'Taylor', 38, 'Architect', 'Portland'),
+    ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+people_records.insert(0, ('Nastya', 'Chuhaieva', 27, 'QA engineer', 'Dnipro'))
+print(people_records)
+people_records[1], people_records[5] = people_records[5], people_records[1]
+print(people_records)
+condition = people_records[6][2] and people_records[10][2] and people_records[13][2] > 30
```

Comment:
```suggestion
condition = (people_records[6][2] > 30) and (people_records[10][2] > 30) and (people_records[13][2] > 30)
```
but the best option will be to use 
`condition = all(record[2] > 30 for record in ( people_records[6], people_records[10], people_records[13])) `